### DISCUSSION TITLE
How to access symlinked package in `node_modules`

### DISCUSSION THREAD

**Discussion by rhlsthrm** [vercel/turborepo#741] [Help]
I have a monorepo where one package (A) only generates JSON files in a directory called `out`. My other package (B) has A as a dependency. I can import from the package in my TS code with `import X from "@org/A/out/X.json`. However, in this particular setup, I need to add the directory with the JSON files to a config file. So the config file needs something like:
```
{
  "external": "node_modules/@org/A"
}
```

This isn't working for me with the turbo setup, and inside `node_modules` inside B I don't see the symlink to A. Am I doing something wrong or misunderstanding how turbo works?

**Comment by anthonyshew** [comment]
`turbo` wouldn't prevent this behavior. 🤔 

Figuring that this was likely a Node.js exporting/packaging error of yore.
